trie/pruning_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 The go-ethereum Authors
```

Comment:
Was it your intention to check this file in?

Reply:
Nope, accident, fill fix.